darts/datasets/__init__.py

```diff
@@ -611,7 +611,7 @@ def pre_proces_fn(extracted_dir, dataset_path):
 
                 output_dict[locationID] = count_series
             output_df = pd.DataFrame(output_dict)
-            output_df.to_csv(dataset_path, line_terminator="\n")
+            output_df.to_csv(dataset_path, lineterminator="\n")
```

Comment:
Actually, can we not just drop the `lineterminator` in general? Then we can avoid the pandas version issues.

Reply:
I guess so but did not try it. If a test cateches it anyways, feel free to apply this suggestion.

```suggestion
            output_df.to_csv(dataset_path)
```

Reply:
I tested locally and it seems to work.